### ISSUE TITLE
[Feature Request] Support for Tags

### ISSUE THREAD

**Issue by tobru** [cloudscale-ch/terraform-provider-cloudscale#35]
It would be cool if this Terraform provider would support specifying [tags](https://www.cloudscale.ch/en/api/v1#tags) on resources.

**Comment by davidhalter** [comment]
Thanks for the request @tobru. We already have a ticket open internally for this. I will try to bring it up in the next sprint planning.

I personally doubt that this is going to happen in Q4 this year.

**Comment by plandolt** [comment]
I'd love to see tag support in terraform. It would be ideal to make groups open on the ansible inverntory plugin.

**Comment by davidhalter** [comment]
> It would be ideal to make groups open on the ansible inventory plugin.

What do you mean by that?

**Comment by plandolt** [comment]
@davidhalter I created the services with terraform and want to provision the services with ansible. Using the inventory plugin for ansible I wanted to make dynamic groups (e.g. all web-workers, all db-servers, etc.). The easiest way would be to make those groups as you illustrated in the examples here (https://docs.ansible.com/ansible/latest/collections/cloudscale_ch/cloud/inventory_inventory.html) using tags. But with the terraform provider, there is no support for setting tags to resources (or at least it's not documented).

**Comment by davidhalter** [comment]
Hey Patrick

Sorry for not answering sooner here. I should have told you that I wanted to bring it up for our next sprint discussions. In general feel free to ping us again if we do not answer.

We decided that this is important enough and I have started working on it. It will probably take a few weeks, since it is not the only thing I'm working on, but I'm confident that this will be in the next release in May.
